packages/@vue/cli-ui/README.md

```diff
@@ -1,15 +1,43 @@
 # @vue/cli-ui
 
-Start development version:
+### Local development
+
+Before starting a local cli-ui instance, 
+consider following the [contributing guide](https://github.com/vuejs/vue-cli/blob/dev/.github/CONTRIBUTING.md) 
+in order to download all required dependencies of vue-cli's packages.
+ 
+Just after, you should build `@vue/cli-ui-addon-webpack` by running:
+```bash
+cd ../cli-ui-addon-webpack
+yarn build
+cd -
+```
+
+Then you should start a local Apollo server:
+
+```
+yarn run apollo
+```
+
+And then in another terminal, you should serve cli-ui:
 
 ```
 yarn run serve
 ```
 
-In another terminal:
+#### Testing
+
+Before running E2E tests, you should start a new local testing apollo server with the following command:
 
 ```
-yarn run apollo
+yarn run test:e2e
+```
+
+Then in another terminal:
```

Comment:
`yarn run test:e2e` should be enough. It both starts the apollo server and the vue-cli e2e service (`yarn run test:e2e:dev`).

Reply:
It seems not.

When running `yarn run test:e2e` alone, it just starts an Apollo server and waits (for nothing).
I have to run both command `yarn run test:e2e` and `yarn run test:e2e:dev` to make tests work.

Maybe this is a bug?

Reply:
Make sure `http://localhost:4040` works. It should start `test:e2e:dev` script as soon as `http://localhost:4040` responds.

Reply:
Yep it works, it show me a GraphQL Playground webapp.

This is my output:

```
➜  cli-ui git:(chore/docs-cli-ui) yarn run test:e2e                  
yarn run v1.7.0
$ yarn run test:clear && start-server-and-test apollo:run:test http://localhost:4040 test:e2e:dev
$ rimraf ../../test/cli-ui-test && rimraf ./live-test
starting server using command "npm run apollo:run:test"
and when url "http://localhost:4040" is responding
running tests using command "test:e2e:dev"

> @vue/cli-ui@3.0.1 apollo:run:test /home/kocal/Dev/vue-cli/packages/@vue/cli-ui
> cross-env VUE_CLI_UI_TEST=true VUE_APP_GRAPHQL_PORT=4040 VUE_APP_CLI_UI_URL=ws://localhost:4040/graphql VUE_CLI_IPC=vue-cli-test vue-cli-service apollo:watch

$ cross-env VUE_CLI_DEBUG=true VUE_CLI_IPC=vue-cli-dev vue-cli-service apollo:run --delay
✔️  GraphQL Server is running on http://localhost:4040/graphql
```

Reply:
So there is something strange here

Reply:
I'm gonna open a new issue and apply your changes